signer/src/network/mod.rs

```diff
@@ -41,4 +138,68 @@ mod tests {
 
         testing::network::assert_clients_can_exchange_messages(client_1, client_2).await;
     }
+
+    #[tokio::test]
+    async fn two_clients_should_be_able_to_exchange_messages_given_a_libp2p_network() {
+        clear_env();
+
+        let _ = tracing_subscriber::fmt()
+            .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+            .try_init();
```

Comment:
I don't think we should have this in the tests, since I believe it installs a global subscriber. I mean, I do this locally when developing but remove it in the PR. It's only an issue if we have any other tests with logging turned on though.

Reply:
Agreed (though it is nice to have when debugging why something went to crap). The global subscriber is only an issue with `cargo test` btw, `cargo nextest` makes this a non-issue (hint hint :wink: )

Removed in [7173ec1](https://github.com/stacks-network/sbtc/pull/473/commits/7173ec1c34c181344ea16407bb81ccf4e4c260ad)

Reply:
lol, I'm sold on `nextest`. I'll ask in the channel, I don't anticipate any objections (I don't know why I haven't already, maybe you already did but didn't get much feedback?).

Reply:
> maybe you already did but didn't get much feedback?

Yeah I mentioned it in the channel but it was pretty much :cricket:s iirc :sweat_smile: